include/aspect/geometry_model/chunk.h

```diff
@@ -108,21 +108,66 @@ namespace aspect
         virtual
         Point<dim> representative_point(const double depth) const;
 
+        /**
+               * Return the longitude at the western edge of the chunk
```

Comment:
Does astyle really fix up comment indentation? In any case, @bobmyhill , can you just adjust this by hand?